PureBasicIDE/AutoComplete.pb

```diff
@@ -1220,6 +1228,13 @@ Procedure AutoCompleteWindowEvents(EventID)
       Select EventGadgetID
           
         Case #GADGET_AutoComplete_List
+          ;- Added by mk-soft; Date 29.05.2020; Linux Redraw TreeView
```

Comment:
This comment is no longer necessary because Git stores this information about who added, changed, and removed which line.